src/cheri_insts.sail

```diff
@@ -574,6 +574,15 @@ function clause execute (CTestSubset(rd, cb, ct)) =
   RETIRE_SUCCESS
 }
 
+union clause ast = CSetEqualExact : (regidx, regidx, regidx)
+function clause execute (CSetEqualExact(rd, cb, ct)) =
+{
+  let cb_val = readCapRegDDC(cb);
```

Comment:
As I mentioned in https://github.com/CTSRD-CHERI/qemu/pull/121, we may want to use CNULL?